src/Geometry.h

```diff
@@ -131,8 +139,7 @@ class Geometry {
   boundaryType getMaxXBoundaryType();
   boundaryType getMinYBoundaryType();
   boundaryType getMaxYBoundaryType();
-  boundaryType getMinZBoundaryType();
-  boundaryType getMaxZBoundaryType();
```

Comment:
Could these routines still be useful (albeit, perhaps more so in the future) even if we only do xy-planar ray tracing on some z level?